content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -92,7 +93,7 @@ This will be used to access the `local` pilot securely using the ingress gateway
     {{< text bash >}}
     $ kubectl create --context=$CTX_LOCAL ns istio-system
     $ kubectl create --context=$CTX_LOCAL secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem --from-file=samples/certs/cert-chain.pem
-    $ kubectl apply --context=$CTX_LOCAL -f install/kubernetes/helm/istio/templates/crds.yaml
+    $ helm install install/kubernetes/helm/istio-init --name istio-init
```

Comment:
Which context is `helm install` working in? This command implies some required setup that was not mentioned in the earlier instructions - e.g., tiller should be running in the local cluster

Reply:
Right. How do we install CRDs without Helm? Latest releases don't have a `crds.yaml` in them anymore.

Reply:
Good question. I think we do need a non-helm way to install the CRDs.

@andraxylia Do you know what the non-helm plan is?

Reply:
You could do it via a loop to install CRDs one by one.  See https://preliminary.istio.io/docs/setup/kubernetes/helm-install/#option-1-install-with-helm-via-helm-template

However using istio-init is also valid which is why it was created in the first place.